official/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.pyc
```

Comment:
Should this be in the top-level .gitignore?

Reply:
*.pyc is already covered in the top-level .gitignore. The others are all specific to this folder. I'm not sure why we have "env/" or "*.ipynb" listed here though.

Reply:
Cleaned up the file to only include things specific to this folder.